src/Management/Generator/ClassesGenerator.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+/**
+ * This file is part of the contentful-management.php package.
+ *
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\Management\Generator;
+
+use Contentful\Management\Client;
+use Contentful\Management\Query;
+use Contentful\Management\Resource\ContentType;
+use PhpParser\Comment;
+use PhpParser\Node;
+use PhpParser\PrettyPrinter\Standard as StandardPrettyPrinter;
+
+/**
+ * ClassesGenerator class.
+ */
+class ClassesGenerator
+{
+    /**
+     * @var Client
+     */
+    private $client;
+
+    /**
+     * @var EntryGenerator
+     */
+    private $entryGenerator;
+
+    /**
+     * @var MapperGenerator
+     */
+    private $mapperGenerator;
+
+    /**
+     * ClassesGenerator constructor.
+     *
+     * @param Client $client
+     */
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+        $defaultLocale = 'en-US';
+
+        $locales = $this->client->locale->getAll();
+        foreach ($locales as $locale) {
+            if ($locale->isDefault()) {
+                $defaultLocale = $locale->getCode();
+
+                break;
+            }
+        }
+
+        $this->entryGenerator = new EntryGenerator($defaultLocale);
+        $this->mapperGenerator = new MapperGenerator($defaultLocale);
+    }
+
+    /**
+     * @param string $namespace
+     *
+     * @return array
```

Comment:
Yeah, some IDEs work better with these annotations rather than with "regular" type hint. Doesn't cost me anything to write them (I can mostly autogenerate them), plus my editor complains if I don't have them 😁

Reply:
So maybe to make them more relevant, can you sprinkle in some more information? What are those parameters used for? What are the boundaries, what happens when I put in a `null`? Stuff a user would expect ...

Reply:
This is not Java, if I say `string $namespace`, passing `null` will cause an error 😉